markdown/core.py

```diff
@@ -442,18 +445,14 @@ def convertFile(
                 output_file.close()
             else:
                 writer = codecs.getwriter(encoding)
-                output_file = writer(output, errors="xmlcharrefreplace")
-                output_file.write(html)
+                output_writer = writer(output, errors="xmlcharrefreplace")
+                output_writer.write(html)
```

Comment:
Not sure why this change was made at all. Note that the [Contributing Guide](https://python-markdown.github.io/contributing/#code-style-guide) states:

> Legacy code which does not follow the guidelines should only be updated if and when other changes (bug fix, feature addition, etc.) are being made to that section of code. While new features should be given names that follow modern Python naming conventions, existing names should be preserved to avoid backward incompatible changes.

I realize that in this instance the variable name is not exposed outside of this method so there is no concern over a backward incompatible change, but the general principle remains. Please, let's refrain from making unnecessary changes just for personal preference.

Reply:
I embrace Python not being strongly typed. If that means making changes like this, then no thank you.